backend/src/main/kotlin/blue/mild/covid/vaxx/service/PatientService.kt

```diff
@@ -121,11 +124,11 @@ class PatientService(
             logger.debug { "Saving registration." }
             patientRepository.savePatient(
                 id = entityId,
-                firstName = registration.firstName.trim(),
-                lastName = registration.lastName.trim(),
-                phoneNumber = registration.phoneNumber.trim(),
-                personalNumber = normalizePersonalNumber(registration.personalNumber),
-                email = registration.email.trim().toLowerCase(),
+                firstName = registration.firstName.removeAllWhitespaces(),
```

Comment:
co kdyz ma nekdo prostredni jmeno a napise ho sem? nebo do lastname? to pak znamena, ze ho ulozime blbe